docs/apis/avalanchego/public-api-server.md

```diff
@@ -60,5 +85,21 @@ If you have questions, problems or suggestions, come [talk to us](https://chat.a
 
 :::warning Disclaimer
 
-Ava Labs offers no warranty/guarantee on the quality or uptime of these 3rd party RPC providers
-:::
\ No newline at end of file
+Ava Labs offers no warranty/guarantee on the quality or uptime of these community RPC providers
+:::
+
+### Ankr
+
+- For C-Chain API, the URL is `https://rpc.ankr.com/avalanche`.
+- For X-Chain API, the URL is `https://rpc.ankr.com/avalanche-x`.
+- For P-Chain API, the URL is `https://rpc.ankr.com/avalanche-p`.
+- For Fuji Testnet API, the URL is `https://rpc.ankr.com/avalanche_fuji`.
```

Comment:
then what are the endpoints for X/P/C-chain API on Fuji? I think we should list them out.

Reply:
Issue resolved